Dockerfile

```diff
@@ -0,0 +1,111 @@
+ARG BASE_IMAGE=ubuntu:20.04
+FROM ${BASE_IMAGE}
+
+# TODO: consider using iterative.ai or something else
+LABEL maintainer="dvc.org"
+
+# CONFIGURE NON-INTERACTIVE APT
+ENV DEBIAN_FRONTEND=noninteractive
+RUN echo 'APT::Get::Assume-Yes "true";' > /etc/apt/apt.conf.d/90assumeyes
+
+# CONFIGURE SHELL
+SHELL ["/bin/bash", "-c"]
+
+# INSTALL CORE DEPENDENCIES
+RUN : \
```

Comment:
don't really like this `:` thing

Reply:
It's just a hack to keep all the lines with the `&& command \` syntax so they're interchangeable and replaceable. 🙃  What would you suggest instead?

Reply:
yes I know but the previous version was fine and managed to keep things aligned. If you want a hybrid you could use my style (one space indent, or more accurately ` &&` continuation):

```bash
...
RUN add-apt-repository ppa:git-core/ppa --yes \
 && add-apt-repository ppa:longsleep/golang-backports --yes \
 && curl --location https://deb.nodesource.com/setup_12.x | bash \
...
```

Reply:
🧙‍♂️ Awesome! Definitely better than my [lapalissade](https://en.wikipedia.org/wiki/Lapalissade) and, in my opinion, much better than the original.

Reply:
well if/when we add a linter for these files it will become consistently ugly anyway :D

Reply:
_Consistently ugly_ can be regarded as a technical oxymoron. If we lint them and catch early any errors derived from edition, style would be a minor issue.